consensus/propagation/peer_state_test.go

```diff
@@ -18,96 +17,6 @@ func newTestPeerState() *PeerState {
 	return newPeerState(&peer, log.NewNopLogger())
 }
 
-func TestPeerState_SetHaves(t *testing.T) {
```

Comment:
these tests were basically just testing the bitmaps, so I felt like they added more tech debt than value.